internal/observability/events/eventer_retry.go

```diff
@@ -0,0 +1,79 @@
+package event
+
+import (
+	"context"
+	"fmt"
+	"math"
+	"math/rand"
+	"time"
+
+	"github.com/hashicorp/boundary/internal/errors"
+	"github.com/hashicorp/eventlogger"
+	"github.com/hashicorp/go-multierror"
+)
+
+const (
+	// stdRetryCount is the standard number of times for retry when sending events
+	stdRetryCount = 3
+)
+
+type backoff interface {
+	duration(attemptNumber uint) time.Duration
+}
+
+type expBackoff struct{}
+
+func (b expBackoff) duration(attempt uint) time.Duration {
+	r := rand.Float64()
+	return time.Millisecond * time.Duration(math.Exp2(float64(attempt))*5*(r+0.5))
```

Comment:
Might be nice to have a comment here or on the method

Reply:
done.